pkgs/servers/monitoring/zabbix/agent2.nix

```diff
@@ -46,8 +46,13 @@ import ./versions.nix ({ version, sha256 }:
     '';
 
     installPhase = ''
+      mkdir -p $out/sbin
+
       install -Dm0644 src/go/conf/zabbix_agent2.conf $out/etc/zabbix_agent2.conf
       install -Dm0755 src/go/bin/zabbix_agent2 $out/bin/zabbix_agent2
+
+      # create a symlink which is compatible with the zabbixAgent module
+      ln -s $out/bin/zabbix_agent2 $out/sbin/zabbix_agentd
```

Comment:
```suggestion
      ln -s $out/bin/zabbix_agent2 $out/bin/zabbix_agentd
```

Reply:
The module is specifically looking for `sbin/`:
https://github.com/NixOS/nixpkgs/blob/d25ea6a0d2a847fb52131da546f2a866656fbafa/nixos/modules/services/monitoring/zabbix-agent.nix#L163

Reply:
We can't just update that path, can we?

Reply:
We could definitely update the module to point to `/bin` instead of `/sbin`. We would just need to make sure we update all our packages accordingly. If anyone was using a custom or older package that might break on them... but we don't support that generally anyways, and they would figure it out quickly I'm sure.

Given the above, let me know if you want me to amend the PR and I'm happy to do so.

Reply:
I thought about this more and there is a possiblity of breakage so I think the change you suggest should go into a future PR. The intent is to fix the stable release with this.